.github/workflows/ci_linux_x64-libshortfin.yml

```
@@ -46,6 +47,7 @@ jobs:
         repository: iree-org/iree
         path: ${{ env.IREE_REPO_DIR }}
         submodules: false
+        ref: ${{ env.IREE_REF }}
```

Any particular reason to store this in an environment variable instead of inlining it?

```suggestion
        ref: candidate-20240904.1006
```

Unless using it in multiple places, I'd prefer to save the one line of code :D

---

The intention was to make it more convenient if this (as well as other vars) to have them grouped and near to the top if those need to be updated. But no strong feeling.

---

Gotcha. I see your point, but I also like to having some of that local to the `actions/checkout` step. Not a big deal with way.

---

I removed the environment variable and moved it to the action :)